Docker/generate_signal_docker_test.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import nibabel as nib
+from utilities.data_simulation.GenerateData import GenerateData
+
+def save_nii(data, filename='ivim_image.nii.gz'):
```

Comment:
There's already a `save_nifti_file` in the wrapper.

Reply:
Yeah, sure.